src/components/ButtonPanel/ButtonPanel.js

```diff
@@ -6,38 +6,43 @@ const buttonPanel = ({ clickHandler }) => {
 
   return (
     <div>
-      <div className = "group group-1">
+
+    // <div className = "button" style = {style} onClick ={handleClick}>
+    //   <h3>{props.name}</h3>
+    // </div>
+
+      <div className = "group group-1 ">
         <Button name = "AC" color = "#E0E0E0" clickHandler = {clickHandler}/>
         <Button name = "+/-" color = "#E0E0E0" clickHandler = {clickHandler}/>
         <Button name = "%" color = "#E0E0E0" clickHandler = {clickHandler}/>
         <Button name = "/" clickHandler = {clickHandler}/>
       </div>
 
-      <div className = "group group-2">
+      <div className = "group group-2 ">
         <Button name = "7" color = "#E0E0E0" clickHandler = {clickHandler}/>
         <Button name = "8" color = "#E0E0E0" clickHandler = {clickHandler}/>
         <Button name = "9" color = "#E0E0E0" clickHandler = {clickHandler}/>
         <Button name = "X" clickHandler = {clickHandler}/>
       </div>
 
 
-      <div className = "group group-3">
+      <div className = "group group-3 ">
         <Button name = "4" color = "#E0E0E0" clickHandler = {clickHandler}/>
         <Button name = "5" color = "#E0E0E0" clickHandler = {clickHandler}/>
         <Button name = "6" color = "#E0E0E0" clickHandler = {clickHandler}/>
         <Button name = "-" clickHandler = {clickHandler}/>
       </div>
 
 
-      <div className = "group group-4">
+      <div className = "group group-4 ">
```

Comment:
Please create your buttons dynamically by using two for-loops, where the first one iterates over the groups you have and the second one iterates over every button in the particular group.